reflex/.templates/apps/blank/code/blank.py

```diff
@@ -16,19 +16,33 @@ def index() -> rx.Component:
     return rx.container(
         rx.color_mode.button(position="top-right"),
         rx.vstack(
-            rx.heading("Welcome to Reflex!", size="9"),
+            rx.heading("Welcome to Reflex 🦕", size="9"),
```

Comment:
i had three team members here in the office take a look at the new layout and they all said "why is there a dinosaur"

Reply:
I added a dinosaur to make the heading fancy, I can change it to "✨" if you want to.